helpers/events_filter/events_filter.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime, timedelta
+from graphql import GraphQLError
+import pytz
+
+from api.events.models import Events as EventsModel
+
+utc = pytz.utc
+
+
+def filter_events_by_date_range(query, start_date, end_date):
+    """
+    Return events that  fall in the date range
+    """
+    if start_date and not end_date:
+        raise GraphQLError("endDate argument missing")
+    if end_date and not start_date:
+        raise GraphQLError("startDate argument missing")
+    if not start_date or None and not end_date or None:
```

Comment:
@koitoror What is the purpose of `or None` here?

Reply:
its to cover for an edge case for no end_date is supplied as a param

Reply:
are you sure it is necessary?

Reply:
upon testing i think i agree the implementation is not necessary since error messages are enough to capture the same.